doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2696,6 +2696,36 @@ Bluetooth samples
 
 .. rst-class:: v3-0-2 v3-0-1 v3-0-0
 
+NCSDK-34582: The :ref:`fast_pair_locator_tag` sample uses incorrect partition map for the nRF54L10 board target
```

Comment:
Should we have ``nrf54l15dk/nrf54l10/cpuapp`` board target or nRF54L10 SoC here?

Reply:
We can use the "nRF54L10 SoC" term to make the issue title shorter